docs/usage/models.md

```diff
@@ -0,0 +1,241 @@
+# Unity Catalog Registered Models and Model Versions
+
+This page shows you how to use Unity Catalog to store, access and govern Registered Models and Model Versions..
+
+Registered Models are logical containers for ML models within Unity Catalog.  A registered model is comprised of any
+number of Model Versions which represent different iterations of the model that you with to keep track of from within Unity Catalog.
+
+Unity catalog provides all of the functionality needed to keep track of model in your model development workflow.
+
+The following diagram shows an example of a Unity Catalog instance with a registered model: `iris_regression` that contains 1 model version:
+
+![UC Functions](../assets/images/uc_functions.png)
```

Comment:
@dennyglee @MrPowers We need a similar image created for this link to show how registered models and model versions work in UC.